### ISSUE TITLE
Some documents can have different copyrights

### ISSUE THREAD

**Issue by deniak** [w3c/specberus#1716]
Some documents may have different copyrights. This should of course be approved by @plehegar and the team contacts of the said WG/IG.
At the moment, the only way for these documents to pass pubrules is to add the copyright rule as an [exception for specific specs](https://github.com/w3c/specberus/blob/main/lib/exceptions.json) but it means we do not check the copyright.
Instead, we should have the list of expected copyrights in a separate json so we can make sure the documents are using the right one.

@iherman provided the list of epub documents that have a different copyright:
- epub-overview-33
- epub-33
- epub-rs-33
- epub-multi-rend-11
- epub-tts-10
- epub-ssv-11
- epub-a11y-11
- epub-a11y-tech-11
- epub-aria-authoring-11
- epub-fxl-a11y
- epubcfi or epubcfi-11

@iherman, @mattgarrish, for these documents, can you confirm the copyright should be:
```html
<p class="copyright">
    <a href=["https://www.w3.org/Consortium/Legal/ipr-notice#Copyright"](https://www.w3.org/Consortium/Legal/ipr-notice#Copyright)>Copyright</a>&nbsp;©&nbsp;1999-2023
    <a href=["https://www.idpf.org"](https://www.idpf.org/)>International Digital Publishing Forum</a>
    and
    <a href=["https://www.w3.org/"](https://www.w3.org/)>World Wide Web Consortium</a>.
    <abbr title="World Wide Web Consortium">W3C</abbr><sup>®</sup>
    <a href=["https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer"](https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer)>liability</a>,
    <a href=["https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks"](https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks)>trademark</a> and
    <a rel="license" href=["https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document) title="W3C Software and Document Notice and License">permissive document license</a> rules apply.
    </p>
```

**Comment by iherman** [comment]
Yes, I confirm the list of the documents as well as the alternate copyright text. Thank you.

**Comment by mattgarrish** [comment]
Sounds right to me. Not sure that CFI will ever see the light of day, as that was only resurrected as part of the locator's work, which seems to have abandoned it. But I suppose better to include it and not have to make changes later.

**Comment by iherman** [comment]
@deniak @plehegar @jennyliang220 can you tell us when this will be completed? We have a new PR waiting to me merged soon, and they will all hit the same issue through echidna...

**Comment by deniak** [comment]
@iherman, @jennyliang220 is working on this but it's taking more time than expected. Once the PR is ready, I'll review and deploy it but I'm off until the 24th so I don't expect this to be merged before that.
If you need to publish the doc ASAP, please sync with @jennyliang220 so she can publish manually.

**Comment by iherman** [comment]
@deniak ack.

@mattgarrish I presume we can wait with PR merges until next week or, more exactly, we are fine with notes not being officially published by then, right?

I have (again...) some medical issues to handle next week, I may be less available than usual, b.t.w.

Thanks Denis.

**Comment by mattgarrish** [comment]
> I presume we can wait with PR merges until next week or, more exactly, we are fine with notes not being officially published by then, right?

Ya, that doesn't sound like a problem to me.

**Comment by plehegar** [comment]
ping on this front. We're looking at publishing documents on the 11h of April so it would be good for this to be resolved before that.

**Comment by deniak** [comment]
I'll sync with @jennyliang220 and will provide an ETA tomorrow but we'll make sure it's ready by next week.

**Comment by deniak** [comment]
@iherman @plehegar, the update to pubrules has not been deployed.
The copyright for the epub specs is now accepted, see [pubrules results for CR-epub-33-20230221](https://www.w3.org/pubrules/?url=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FCR-epub-33-20230221%2F&profile=CR&validation=simple-validation&informativeOnly=false&echidnaReady=false&patentPolicy=pp2020)

**Comment by deniak** [comment]
Sorry, it *has* been deployed!

**Comment by iherman** [comment]
I just realize, while doing some changes on the documents themselves: does the text checked by specberus checks the exact dates? Ie, does it check

```
&nbsp;©&nbsp;1999-2023
```

or something like

```
&nbsp;©&nbsp;1999-YEAROFPUBLICATION
```

Of course, fixed dates work for today, but would be better to avoid looking at this thing again if we do a change on the specs in 2024...

**Comment by deniak** [comment]
> I just realize, while doing some changes on the documents themselves: does the text checked by specberus checks the exact dates? Ie, does it check
> 
> ```
> &nbsp;©&nbsp;1999-2023
> ```
> 
> or something like
> 
> ```
> &nbsp;©&nbsp;1999-YEAROFPUBLICATION
> ```
> 
> Of course, fixed dates work for today, but would be better to avoid looking at this thing again if we do a change on the specs in 2024...

The latter. See https://github.com/w3c/specberus/blob/main/lib/copyright-exceptions.json
